libs/portals/my-pages/signature-collection/src/screens/Municipal/OwnerView/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { ActionCard, Stack, Text, Box } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../../lib/messages'
+import Managers from '../../shared/Managers'
+import { SignatureCollection, SignatureCollectionList } from '@island.is/api/schema'
+
+const OwnerView = ({
+  currentCollection,
+  lists
+}: {
+  currentCollection: SignatureCollection
+  lists: SignatureCollectionList[]
+}) => {
+  const { formatMessage } = useLocale()
+
+  return (
+    <Stack space={6}>
+      <Box>
+        <Text variant="h4" marginBottom={3}>
+          {formatMessage(m.myListsDescription)}
+        </Text>
+        <ActionCard
+          backgroundColor="white"
+          heading="Borgarbyggð"
```

Comment:
Assuming that these action cards are hard coded for now and this is still feature flagged :+1: